src/plugins/discover/public/hooks/use_navigation_props.tsx

```diff
@@ -85,18 +83,14 @@ export const useNavigationProps = ({
     () => getCurrentBreadcrumb(history?.location?.search),
     [history?.location?.search]
   );
-  const contextSearchHash = useMemo(
-    () => getContextHash(columns, filterManager),
-    [columns, filterManager]
```

Comment:
We could keep the variable and set `const contextSearchHash = getContextHash(columns, filterManager)`

Reply:
100% right, thx, changed it 👍